SYCL/DeviceLib/sycl_ext_intel_math_fp16.cpp

```diff
@@ -0,0 +1,72 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// RUN: %clangxx -fsycl -fno-builtin -fsycl-device-lib-jit-link %s -o %t.out
```

Comment:
I think it can be useful for the debugging if test binary is not overridden.
```suggestion
// RUN: %clangxx -fsycl -fno-builtin -fsycl-device-lib-jit-link %s -o %t.noblt.out
```
This applies to other tests as well.

Reply:
Done.